lttoolbox/transducer.cc

```diff
@@ -503,7 +503,8 @@ Transducer::isEmpty(int const state) const
 void
 Transducer::write(FILE *output, int const decalage, bool write_weights)
 {
-  Compression::multibyte_write(initial, output);
```

Comment:
@TinoDidriksen @ftyers Can you please review this new write and read methods, it is giving corpus test results similar to when we were writing to every binary file.